cfgmgr/vxlanmgr.cpp

```diff
@@ -526,5 +536,38 @@ bool VxlanMgr::deleteVxlan(const VxlanInfo & info)
     return true;
 }
 
-
+void VxlanMgr::clearAllVxlanDevices()
+{
+    std::string stdout;
```

Comment:
no need `std::` in cpp files.

Reply:
I think an explicit namespace(like std) is a good habit, because it can avoid some name conflict. For example, we use **regex** class which was both existed in **std** and **boost**. An explicit declaration, **std::regex**, will make it to be clearer which library we want to use, I think,